yarn-project/circuit-types/src/l2_block.test.ts

```diff
@@ -5,8 +5,8 @@ describe('L2Block', () => {
   it('can serialize an L2 block with logs to a buffer and back', () => {
     const block = L2Block.random(42);
 
-    const buffer = block.toBufferWithLogs();
-    const recovered = L2Block.fromBufferWithLogs(buffer);
+    const buffer = block.toBuffer(true, true);
```

Comment:
Again, think this would be better having a buffer for the block and for the body, because this combination is confusing.